sale_order_field_advanced/models/sale_order_line.py

```diff
@@ -0,0 +1,72 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models, fields, api, _
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    adv_reference = fields.Char()
+    adv_is_tailored = fields.Boolean()
+    adv_characteristics = fields.Text()
+    adv_modification_id = fields.Many2one('sale.order.line.modification')
+    adv_note = fields.Text()
+    adv_complement_id = fields.Many2one('product.product')
+    adv_manufacturing_state_id = fields.Many2one('sale.order.line.process.status',
+                                                 compute='_compute_adv_manufacturing_state_id', store=True)
+    adv_date_order = fields.Datetime(
+        related="order_id.date_order",
+        store=True,
+    )
+    adv_product_has_process_control = fields.Boolean(
+        related="product_id.categ_id.has_process_control",
+    )
+
+    adv_manufacturing_state_name = fields.Char(
+        string='Manufacturing State Name',
+        related='adv_manufacturing_state_id.name',
+    )
+
+    adv_manufacturing_state_decoration = fields.Char(compute='_compute_adv_manufacturing_state_decoration')
+
+    def _compute_adv_manufacturing_state_decoration(self):
```

Comment:
Even for a compute method that doesn't store data `api.depends` decorator should be defined when possible:
```suggestion
    @api.depends("adv_manufacturing_state_id.decoration_color")
    def _compute_adv_manufacturing_state_decoration(self):
```